src/EventSubscriber/GlobalredirectSubscriber.php

```diff
@@ -194,7 +194,7 @@ public function globalredirectForum(GetResponseEvent $event) {
     $term = $this->entityManager->getStorage('taxonomy_term')->load($matches[1]);
     if (!empty($term) && $term->url() != $request->getPathInfo()) {
       $system_path = $this->aliasManager->getPathByAlias(ltrim($term->url(), '/'));
-      $this->setResponse($event, Url::fromUri('user-path:' . $system_path));
+      $this->setResponse($event, Url::fromUri('internal:' . $system_path));
```

Comment:
Possibly...however, all tests are green. We did have to add that slash in commit cb182e9, but I haven't looked at why it isn't needed in the other instance. I think it might already have the slash. Note that the slash gets removed right above where we add it back.